docs/CHANGELOG.md

```diff
@@ -87,6 +87,9 @@ Version 1.2.0 (Willow): Not yet released
 - Fix rare crash when rapidly switching between players in first person spectate mode
 - Fix server processing `af_ping_location_req` packets when feature was turned off
 
+[@is-this-c](https://github.com/is-this-c)
```

Comment:
Please move this under the `Minor features, changes, and enhancements` heading rather than `Bug fixes`